setuptools/command/bdist_egg.py

```diff
@@ -382,8 +382,9 @@ def scan_module(egg_dir, base, name, stubs):
         for bad in [
             'getsource',
             'getabsfile',
+            'getfile',
             'getsourcefile',
-            'getfile' 'getsourcelines',
+            'getsourcelines',
```

Comment:
That was a real bug!

Reply:
Thank you very much @DimitriPapadopoulos for working on this.

Given that this PR for the time being is failing the CI, would you like to extract this bug fix in a PR of its own?

Reply:
I can do that, but what's the intent? As far as I can tell, this change is not (directly) responsible for the CI failures.

Reply:
Great, if the CI is no longer failure that is OK. We can proceed with this PR.